controller/testController.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from controller import Controller
+
+
+class TestController(unittest.TestCase):

```

Comment:
Need unit tests for the controller.

Need integration tests for controller and event handler interaction.